HPXMLtoOpenStudio/resources/defaults.rb

```diff
@@ -1886,6 +1886,21 @@ def self.apply_hvac(runner, hpxml_bldg, weather, convert_shared_systems, unit_nu
       heat_pump.heating_efficiency_hspf2 = nil
     end
 
+    # Convert EER to CEER
```

Comment:
Convert EER to CEER ahead in defaults.rb, instead of internally in hvac.rb, to align with other hvac efficiency preprocessing.

Reply:
Agreed, that's a good change. Thanks!